front/init/db.ts

```diff
@@ -24,4 +24,11 @@ async function main() {
   process.exit(0);
 }
 
-await main();
+main()
```

Comment:
we don't have top-level await in Typescript unless we set our `compilerOptions.target` to `"es2017"`